rust/decentraland-godot-lib/src/scene_runner/scene_manager.rs

```diff
@@ -939,80 +939,69 @@ impl INode for SceneManager {
             &current_pointer_raycast_result,
         );
 
-        let should_update_tooltip = !GodotDclRaycastResult::eq_key(
```

Comment:
Is there any possibility of keeping this?

Reply:
The distance changes on every frame and the eq_key just compare if the scene and entity are the same than the last one